doc/man3/X509_verify_cert.pod

```diff
@@ -28,25 +32,29 @@ OpenSSL internally for certificate validation, in both the S/MIME and
 SSL/TLS code.
 
 A negative return value from X509_verify_cert() can occur if it is invoked
-incorrectly, such as with no certificate set in B<ctx>, or when it is called
-twice in succession without reinitialising B<ctx> for the second call.
+incorrectly, such as with no certificate set in I<ctx>, or when it is called
+twice in succession without reinitialising I<ctx> for the second call.
 A negative return value can also happen due to internal resource problems or if
 a retry operation is requested during internal lookups (which never happens
 with standard lookup methods).
 Applications must check for <= 0 return value on error.
 
+The X509_verify_chain() behaves like X509_verify_cert() except that its
+target certificate defaults to the first the first element of the
+list of untrusted certificates in I<ctx>.
+
 =head1 RETURN VALUES
 
-If a complete chain can be built and validated this function returns 1,
-otherwise it return zero, in exceptional circumstances it can also
-return a negative code.
+Both functions return 1 if a complete chain can be built and validated,
+otherwise they return 0, and in exceptional circumstances (such as malloc
+failure and internal errors) they can also return a negative code.
 
-If the function fails additional error information can be obtained by
-examining B<ctx> using, for example L<X509_STORE_CTX_get_error(3)>.
+On error or failure additional error information can be obtained by
+examining I<ctx> using, for example, L<X509_STORE_CTX_get_error(3)>.
 
 =head1 BUGS
 
-This function uses the header F<< <openssl/x509.h> >>
+X509_verify_cert() is declared in the header F<< <openssl/x509.h> >>
```

Comment:
Just drop this as there is practically no difference between including x509.h or x509_vfy.h as both include each other.

Reply:
That's good news. Done.

So I've taken the opportunity to move
```
int X509_verify_cert(X509_STORE_CTX *ctx);
```
to where it belongs: `x509_vfy.h.in`.